plugins/providers/virtualbox/cap/configure_disks.rb

```diff
@@ -131,15 +126,15 @@ def self.handle_configure_disk(machine, disk, all_disks, controller_name)
           else
             # TODO: What if it needs to be resized?
 
-            disk_info = machine.provider.driver.get_port_and_device(current_disk["UUID"])
+            disk_info = machine.provider.driver.get_port_and_device(current_disk[:uuid])
```

Comment:
This is probably cleaner anyway now that we have access to the port and device. We didn't previously here, so I think this makes sense!

Reply:
Oh hm, I think this needs to change beyond this. The actual issue here is before, we got `all_disks` from `vboxmanage list hdds`, which meant that it could exist in VirtualBox without being attached to a guest. But now that we get `all_disks` from `vboxmanage showvminfo`, which only shows disks attached to the guest. So I think instead we'll have to grab everything from `vboxmanage list hdds` and see if the disk is in there but not in our current `all_disks` data structure